docs/configuration/index.md

```diff
@@ -650,25 +650,25 @@ When you specify an error response transform strategy other than `none`, Druid t
 ###### No error response transform strategy
 
 In this mode, Druid leaves error responses from underlying services unchanged and returns the unchanged errors to the API client.
-This is the default Druid error response mode. To explicitly enable this strategy, set `druid.server.http.errorResponseTransformStrategy.strategy` to "none".
+This is the default Druid error response mode. To explicitly enable this strategy, set `druid.server.http.errorResponseTransform.strategy` to "none".
 
 ###### Allowed regular expression error response transform strategy
 
-In this mode, Druid validates the error responses from underlying services against a list of regular expressions. Only error messages that match a configured regular expression are returned. To enable this strategy, set `druid.server.http.errorResponseTransformStrategy.strategy` to `allowedRegex`.
+In this mode, Druid validates the error responses from underlying services against a list of regular expressions. Only error messages that match a configured regular expression are returned. To enable this strategy, set `druid.server.http.errorResponseTransform.strategy` to `allowedRegex`.
 
 |Property|Description|Default|
 |--------|-----------|-------|
-|`druid.server.http.errorResponseTransformStrategy.allowedRegex`|The list of regular expressions Druid uses to validate error messages. If the error message matches any of the regular expressions, then Druid includes it in the response unchanged. If the error message does not match any of the regular expressions, Druid replaces the error message with null or with a default message depending on the type of underlying Exception. |`[]`|
+|`druid.server.http.errorResponseTransform.allowedRegex`|The list of regular expressions Druid uses to validate error messages. If the error message matches any of the regular expressions, then Druid includes it in the response unchanged. If the error message does not match any of the regular expressions, Druid replaces the error message with null or with a default message depending on the type of underlying Exception. |`[]`|
```

Comment:
```suggestion
|`druid.server.http.errorResponseTransform.allowedRegex`|The list of regular expressions Druid uses to validate error messages. If the error message matches any of the regular expressions, then Druid includes it in the response unchanged. If the error message does not match any of the regular expressions, Druid replaces the error message with null or with a default message depending on the type of underlying Exception. Only applies when `druid.server.http.errorResponseTransform.strategy` is set to `allowedRegex` |`[]`|
```
Would this clarification be too much?

Reply:
I don't think it's needed. This whole section (line 655 to 675) only applies when `druid.server.http.errorResponseTransform.strategy` is set to `allowedRegex`.

Reply:
Otherwise, if we add more properties for the allowed regular expression error response transform strategy then we would have to repeat this message (`Only applies when `druid.server.http.errorResponseTransform.strategy` is set to `allowedRegex``) for all the properties in this table.

Reply:
SGTM! Thanks for the clarification.